cmd/sites.go

```diff
@@ -0,0 +1,158 @@
+	Use:   "sites",
+	Short: "Manage sites",
+	Long:  `List, create, delete and manage sites.`,
+}
+
+type Site struct {
+	CreatedOn     string
+	Id            string `json:"_id"`
+	LastPublished string
+	Name          string
+	PreviewUrl    string
+	ShortName     string
+	Timezone      string
+}
+
+// ListSitesResponse represents a response to the list sites request in Webflow.
+// See: https://developers.webflow.com/reference/list-sites
+type ListSitesResponse []Site
+
+// listSitesCmd represents the command to list sites in Webflow.
+var listSitesCmd = &cobra.Command{
+	Use:   "list",
+	Short: "list sites",
+	Run: func(cmd *cobra.Command, args []string) {
+		c := internal.NewClient(ApiToken)
+		body, err := c.Get([]string{"sites"})
+		if err != nil {
+			log.Fatalf("Request failed: %v", err)
+		}
+
+		var response ListSitesResponse
+		err = json.Unmarshal(body, &response)
+		if err != nil {
+			log.Fatalf("Failed to unmarshal response body: %v", err)
+		}
+		for _, site := range response {
+			fmt.Printf("%s\t%s\t%s\t%s\n", site.Name, site.Id, site.LastPublished, site.PreviewUrl)
+		}
+	},
+}
+
+// GetSitesResponse represents a response to the get site request in Webflow.
+// See: https://developers.webflow.com/reference/get-site
+type GetSitesResponse Site
+
+// getSitesCmd represents the command to get a site in Webflow.
+var getSitesCmd = &cobra.Command{
+	Use:   "get [site_id]",
+	Short: "get a site",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		siteId := args[0]
+		c := internal.NewClient(ApiToken)
+		body, err := c.Get([]string{"sites", siteId})
+		if err != nil {
+			log.Fatalf("Request failed: %v", err)
+		}
+
+		var response GetSitesResponse
+		err = json.Unmarshal(body, &response)
+		if err != nil {
+			log.Fatalf("Failed to unmarshal response body: %v", err)
+		}
+		w := tabwriter.NewWriter(os.Stdout, 1, 1, 1, ' ', 0)
+		fmt.Fprintf(w, "name:\t%s\n", response.Name)
+		fmt.Fprintf(w, "id:\t%s\n", response.Id)
+		fmt.Fprintf(w, "created on:\t%s\n", response.CreatedOn)
+		fmt.Fprintf(w, "last published:\t%s\n", response.LastPublished)
+		fmt.Fprintf(w, "preview url:\t%s\n", response.PreviewUrl)
+		fmt.Fprintf(w, "short name:\t%s\n", response.ShortName)
+		fmt.Fprintf(w, "timezone:\t%s\n", response.Timezone)
+		w.Flush()
+	},
+}
+
+// PublishSiteResponse represents a response to the publish site request in Webflow.
+// See: https://developers.webflow.com/reference/publish-site
+type PublishSiteResponse struct {
+	Queued bool
+}
+
+// publishSitesCmd represents the command to publish a site in Webflow.
+var publishSitesCmd = &cobra.Command{
+	Use:   "publish [site_id]",
+	Short: "publish a site",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		siteId := args[0]
+		c := internal.NewClient(ApiToken)
+		body, err := c.Get([]string{"sites", siteId, "publish"})
+		if err != nil {
+			log.Fatalf("Request failed: %v", err)
+		}
+
+		var response PublishSiteResponse
```

Comment:
It'd be nice to print the response result to make it clear that the cmd finished successfully

Reply:
The response is `"queued": true` which is not particularly helpful in my opinion so thought silence was better.

Reply:
Oh ok, yea that's not a big deal. Alternatively I was thinking of some generic message that says: "Published successfully"

Reply:
But it might not have tho. The command was sent successfully and as per unix philosophy I think if there is no additional information, then the exit code should be enough to indicate if the action was successful.

Reply:
Ah ok, yea, right, queued != published. Fair enough